apps/frontend/src/components/global/checklist/ChecklistSeverityOverride.vue

```diff
@@ -0,0 +1,68 @@
+<template>
+  <v-card class="text-center px-8 pt-2" height="300px">
```

Comment:
probably have it be responsive in size instead of 300px.  maybe make 300 the min height or something?

Reply:
This height was removed